string command
string object_name
string pose
float32 speed_factor
float32 acc_factor
float32 goal_tolerance
